scripts/x-charts-pro.exports.json

```diff
@@ -150,6 +151,22 @@
   { "name": "Direction", "kind": "TypeAlias" },
   { "name": "EMPTY_SIDES", "kind": "Variable" },
   { "name": "FadeOptions", "kind": "TypeAlias" },
+  { "name": "FunnelChartProps", "kind": "Interface" },
+  { "name": "FunnelChartSlotExtension", "kind": "Interface" },
+  { "name": "FunnelChartSlotProps", "kind": "Interface" },
+  { "name": "FunnelChartSlots", "kind": "Interface" },
+  { "name": "FunnelCurveType", "kind": "TypeAlias" },
+  { "name": "FunnelDataPoints", "kind": "TypeAlias" },
```

Comment:
Not sure about what user will do with htis type

Reply:
Alternative is to export it from a different file or export all but 1 type from the `funnel.types` file 😓